src/content/concepts/index.md

```diff
@@ -154,3 +151,16 @@ There are many plugins that webpack provides out of the box! Check out our [list
 Using plugins in your webpack config is straightforward - however, there are many use cases that are worth further exploration.
 
 [Learn more!](/concepts/plugins)
+
+
+## Mode
+
+By choosing between `development` and `production` **mode** you can enable webpack's automatic optimizations according to selected option.
```

Comment:
I would maybe change this to: "By setting the `mode` parameter to either `development` or `production`, you can enable webpack's built-in optimizations that correspond with the selected mode."